src/test/java/core/basesyntax/strategy/IncrementHandlerTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.impl.StorageServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class IncrementHandlerTest {
```

Comment:
Let's add a test case with empty storage;) Also, don't use storageService when you need to add some elements to storage.